GOVERNANCE.md

```diff
@@ -0,0 +1,102 @@
+# Joint Foundation Bootstrap Team
+
+For the current list of Team members, see the project
+[README.md](./README.md).
+
+## Members
+
+The [nodejs/bootstrap](https://github.com/nodejs/bootstrap) GitHub
+repository is maintained by the Team and additional Members who are
+added on an ongoing basis.
+
+## Team Membership
+
+Team Membership is not time-limited. There is no fixed size of the Team.
+
+There is no specific set of requirements or qualifications for Team Membership beyond these rules.
+
+The following groups automatically have membership and can request to be added to the github team:
+
+* Node.js Members: @nodejs/members
+* JSF Project Maintainers
+* JSF TAC
+* Node.js Foundation Board of Directors
+* JSF Board of Directors
+
+## Team Meetings
+
+The Team meets weekly on Zoom.us. A designated moderator
+approved by the Team runs the meeting. Each meeting should be
+published to YouTube.
+
+Items are added to the Team agenda that are considered contentious or
+are modifications of governance, contribution policy, Team membership,
+or release process.
+
+The intention of the agenda is not to approve or review all patches;
+that should happen continuously on GitHub and be handled by the larger
+group of Collaborators.
+
+Any community member or contributor can ask that something be added to
+the next meeting's agenda by logging a GitHub Issue. Any Collaborator,
+Team member or the moderator can add the item to the agenda by adding
+the ***bootstrap-agenda*** tag to the issue.
+
+Prior to each Team meeting the moderator will share the agenda with
+members of the Team. Team members can add any items they like to the
+agenda at the beginning of each meeting. The moderator and the Team
+cannot veto or remove items.
+
+The moderator is responsible for summarizing the discussion of each
+agenda item and sends it as a pull request after the meeting.
+
+## Consensus Seeking Process
+
+The Team follows a
+[Consensus Seeking](http://en.wikipedia.org/wiki/Consensus-seeking_decision-making)
+decision-making model.
+
+When an agenda item has appeared to reach a consensus, the moderator
+will ask "Does anyone object?" as a final call for dissent from the
+consensus.
+
+If an agenda item cannot reach a consensus, a Team member can call for
+to table the issue to the next meeting. The decision must be seconded by
+a majority of the Team or else the discussion will continue.
```

Comment:
I'd like to see a provision in here that if there are multiple choices on the table and consensus cannot be reached on one over the other, *both* choices are documented along with the discussion around it. This will allow us to keep iterating without blocking on any single issue.